cdk/lib/frontend.ts

```diff
@@ -117,71 +114,37 @@ export class Frontend extends GuStack {
       }),
     ];
 
-    const minimumProdInstances = 3;
-
```

Comment:
we no longer need this as we have the `scaling` prop now and can get `scaling.minimumInstances` to get this value